project.py

```diff
@@ -267,6 +267,32 @@ def get_ssn_assignment_prefix():
                 ssn_prefix[int(ls[0])] = ls[1]
     return ssn_prefix
 
+def get_ssn_prefix_count(ssn_array: list) -> dict:

```

Comment:
A minor issue here is that the indentation is not a multiple of 4.  Did you use vscode / GitHub Codespaces when developing this?  I'd be surprised if that didn't default to 4 spaces.